### ISSUE TITLE
[DISCUSSION] Download extra input data from Compute Canada to EC2

### ISSUE THREAD

**Issue by JiaweiZhuang** [geoschem/geos-chem-cloud#30]
## Overview

Currently [our S3 bucket](https://registry.opendata.aws/geoschem-input-data/) only syncs the data on Harvard Odyssey, but not the [extra data on Compute Canada](http://wiki.seas.harvard.edu/geos-chem/index.php/Downloading_GEOS-Chem_data_directories#Compute_Canada_data_directory_archive), notably global high-res metfields. But users can still download data from Compute Canada to EC2 (pulling data into the cloud is free). We should see how fast this is. If it is too slow we should consider adding those data to S3.

The speed can highly depend the AWS region -- I believe that the [Canada region](https://aws.amazon.com/about-aws/whats-new/2016/12/announcing-the-aws-canada-central-region/) would be very fast.

## Action items

- [x] Test the data transfer speed from Compute Canada to EC2 in `us-east-1` and `ca-central-1` regions

**Comment by JiaweiZhuang** [comment]
I got a bandwidth of 10~20 MB/s in both `us-east-1` and `ca-central-1`. Launching EC2 in Canada region does not speed up the download, so 20 MB/s is probably the bandwidth limit forced by Compute Canada. This is 10x slower than downloading from S3 to EC2.

I used this command to download 1-month of GEOS-FP 0.25x0.3125 metfields:

```
wget -r -nH --cut-dirs=2 "http://geoschemdata.computecanada.ca/GEOS_0.25x0.3125/GEOS_FP/2017/07/"
```

1-month of 0.25x0.3125 metfields are ~300 GB. It would take 4 hours to download from Compute Canada and 30 minutes from S3.
